[serget/task1]: initial version of crypto servicve isadded

SergeyGreen7 opened the pull request:
A simple crypto service is added with following features:
- caclulate hash based on sha256
- caclulate hmac based on sha256
- encryption/decryption based on AES algorithm

SergeyGreen7 committed:
[serget/task1]: initial version of crypto servicve isadded

alexandrryazanov commented on src/services/crypto/crypto.service.ts:
```diff
@@ -0,0 +1,35 @@
+    this.secret = secret;
+  }
+
+  getSha256(message: String) {
```
String - это класс встроенный.
string - тип в typescript. 

Странно что не ругнулась IDE.

SergeyGreen7 replied:
у меня VSCode, на который я не устанавливал плагинов, скорее всего поэтому

alexandrryazanov commented on src/services/crypto/crypto.service.ts:
```diff
@@ -0,0 +1,35 @@
+import * as CryptoJS from "crypto-js";
+
+export class MyCrypto {
```
Давай все таки `CryptoService`
Аналогично остальным сервисам

SergeyGreen7 replied:
Ok

alexandrryazanov commented on src/services/crypto/crypto.service.ts:
```diff
@@ -0,0 +1,35 @@
+import * as CryptoJS from "crypto-js";
```
вообще в node уже есть пакет `node:crypto`. Я думал тебе его хватит. Но возможно в crypto-js тебе нужно было что-то. Расскажешь на мите.

SergeyGreen7 replied:
Что первое попалось, то и взял, могу на node:crypto переделать

alexandrryazanov reviewed (changes requested):
Крутяк! 👍 Первый нах)
Пару мелких нюансов написал в комментах. Из серьезного только про `string`

А так красавчик, что разобрался с синтаксисом, разобрался с новой либой и вообще нашел время. Пример салагам 😁

SergeyGreen7 committed:
[serget/task1]:
1. crypto-js is replaced with node:crypto
2. MyCrypto -> CryptoService
3. String -> string
4. MD5 hash caclualtion is added

SergeyGreen7 commented:
Changes based on comments are done

alexandrryazanov reviewed (approved):
LGTM! Approved!
